insights/configs/base.py

```diff
@@ -0,0 +1,57 @@
+""" Configuration class for Insights"""
+
+import logging
+import os
+
+from logging import config
+
+LOGGER = logging.getLogger(__name__)
+
+def get_project_root():
+    """Return the path to the Insights project root directory.
+    :return: A directory path.
+    :rtype: str
+    """
+    return os.path.realpath(os.path.join(
+         os.path.dirname(__file__),
+         os.pardir,
+         os.pardir,
+    ))
+
+class Settings(object):
+    """
+    Insights Settings representation.
+    """
+
+    def __init__(self):
+        """
+        Init for Settings
+        """
+        self._configured = False
+        self._configure_logging()
```

Comment:
_configure_logging() should not be a part of __init__(), is included in the next function itself.